optimizing-memn2n/virtual_env_setup/python3_memn2n_tf_env_cpu_requirements.txt

```diff
@@ -1,31 +1,23 @@
-absl-py==0.7.1
-astor==0.7.1
-bleach==3.3.0
+Keras-Preprocessing==1.1.2
+Werkzeug==2.2.2
+absl-py==1.3.0
 certifi==2022.12.7
-chardet==3.0.4
-gast==0.2.2
-grpcio==1.20.1
-h5py==2.9.0
-html5lib==0.999999999
-idna==2.8
-Keras-Applications==1.0.7
-Keras-Preprocessing==1.0.9
-Markdown==3.1
-mock==2.0.0
-numpy==1.22.0
-pandas==0.23.4
-pbr==5.2.0
-protobuf==3.18.3
-python-dateutil==2.7.5
-pytz==2018.9
-requests==2.21.0
-scikit-learn==0.17.1
-scipy==1.1.0
-sigopt==3.6.0
-six==1.10.0
-tensorboard==1.13.1
-tensorflow>=1.15.4
-tensorflow-estimator==1.13.0
+gast==0.4.0
+grpcio==1.51.1
+h5py==3.7.0
+idna==3.4
+keras==2.10.0
```

Comment:
I believe `keras` is part of TF2 now, so don't need separate req here?

I'm guessing `Keras-Preprocessing` is also included as part of TF2